src/components/FilterItem/FilterItem.tsx

```diff
@@ -37,7 +37,7 @@ const FilterItem: React.FunctionComponent<Props> = ({
 
   return (
     <div>
-      <a onClick={() => toggleFilter(id)}>{activeIcon(id)}</a>
+      <a id={"facet-"+name} onClick={() => toggleFilter(id)}>{activeIcon(id)}</a>
```

Comment:
I think these are called in a loop with the same facet `name`, but different values for the `id` variable.

The `id` attribute on the `a` tag would not be unique.

Reply:
Ohh thanks yeah that makes sense.